test/run-drun/query-footprint.mo

```diff
@@ -0,0 +1,41 @@
+//MOC-FLAG --stable-var-footprint-query=stable-variable-footprint
+
+actor footprint = {
+    stable var s : Nat64 = 42;
+    stable var hello : Text = "Hey You!";
+    var lost : Text = "This will be lost";
+    let e0 = "seed";
+    let e1 = (e0, e0);
+    let e2 = (e1, e1);
+    let e3 = (e2, e2);
+    let e4 = (e3, e3);
+    let e5 = (e4, e4);
+    let e6 = (e5, e5);
+    let e7 = (e6, e6);
+    let e8 = (e7, e7);
+    let e9 = (e8, e8);
+    let e10 = (e9, e9);
+    stable var expl = e10;
```

Comment:
We can go _deeper_. Do we want to?

Reply:
Nope, it gets slow, and at `e13` (dc19ba63e02c113aa28bc0f1c8138714475e9741) the `motoko:stable_types` section overflows the 64kB limit.

Reply:
> to avoid it hitting disk at all

But `e10` isn't occupying a lot of space, or is it?